public/index.html

```diff
@@ -15,6 +15,8 @@
       href="https://fonts.googleapis.com/css2?family=Montserrat:wght@400;700&family=Righteous&display=swap"
       rel="stylesheet"
     />
+    <link href="https://fonts.googleapis.com/icon?family=Material+Icons"
```

Comment:
Loved learning about these icons! Definitely a fun, stylish addition to our project and something to reference for future applications!